src/main/core/main.c

```diff
@@ -28,9 +28,9 @@
 #include "shd-config.h"
 #include "support/logger/logger.h"
 
-static bool _setSchedFifo = false;
+static bool _useSchedFifo = false;
 OPTION_EXPERIMENTAL_ENTRY(
-    "set-sched-fifo", 0, 0, G_OPTION_ARG_NONE, &_setSchedFifo,
+    "set-sched-fifo", 0, 0, G_OPTION_ARG_NONE, &_useSchedFifo,
```

Comment:
Replace the 0 in the arg preceding `G_OPTION_ARG_NONE` to `G_OPTION_FLAG_NONE`. (That makes it more obvious which arg to change if we want to change it to `G_OPTION_FLAG_REVERSE`  later.)

Reply:
Added